src/tests/JIT/HardwareIntrinsics/Arm/Shared/Helpers.cs

```diff
@@ -2293,6 +2293,79 @@ private static long Reduce(Func<long, sbyte, long> reduceOp, sbyte[] op1)
 
         public static int AddPairwiseWideningAndAdd(int[] op1, short[] op2, int i) => (int)(op1[i] + AddWidening(op2[2 * i], op2[2 * i + 1]));
 
+        public static uint AddCarryWideningEven(uint[] op1, uint[] op2, uint[] op3, int i)
+        {
+            uint lsb;
+            ulong res;
+
+            if (i < 0 || i >= op1.Length || i >= op2.Length || i >= op3.Length)
```

Comment:
```suggestion
            if ((i < 0) || (i >= op1.Length) || (i >= op2.Length) || (i >= op3.Length))
```

nit: can you please also wrap the subexpressions below in its own parentheses ? Just like how I did here?